specification/workloadmonitor/resource-manager/Microsoft.WorkloadMonitor/preview/2020-01-13-preview/Microsoft.WorkloadMonitor.json

```diff
@@ -0,0 +1,805 @@
+          "200": {
+            "description": "Search results matching criteria",
+            "schema": {
+              "$ref": "#/definitions/MonitorList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DefaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get monitor list with default values": {
+            "$ref": "./examples/MonitorList_GetDefault.json"
+          },
+          "Get monitor list with filtered values": {
+            "$ref": "./examples/MonitorList_GetFiltered.json"
+          },
+          "Get monitor list with expanded values": {
+            "$ref": "./examples/MonitorList_GetExpanded.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/{resourceType}/{resourceName}/providers/Microsoft.WorkloadMonitor/monitors/{monitorId}": {
+      "get": {
+        "summary": "Get the current status of a monitor of a resource.",
+        "operationId": "MonitorInfo_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/resourceType"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "#/parameters/monitorId"
+          },
+          {
+            "$ref": "#/parameters/expand"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "search results matching criteria",
+            "schema": {
+              "$ref": "#/definitions/Monitor"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DefaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get monitor with default values": {
+            "$ref": "./examples/Monitor_GetDefault.json"
+          },
+          "Get monitor with expanded values": {
+            "$ref": "./examples/Monitor_GetExpanded.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/{resourceType}/{resourceName}/providers/Microsoft.WorkloadMonitor/monitors/{monitorId}/history": {
+      "get": {
+        "summary": "Get history of a monitor of a resource (with optional filter).",
+        "operationId": "MonitorStateChanges_List",
```

Comment:
Likely this shoudkl be 'Monitors_ListStateChanges'

Reply:
Changed to MonitorHistory_ListStateChanges?